### DISCUSSION TITLE
[Observation] Likes / +1s

### DISCUSSION THREAD

**Discussion by ggolcher** [AKASHAorg/akasha-core#1121] [Ideas]
Several customers have expressed a desire to rapidly express agreement or inclination for something, in the form of a Like or a +1 or some other way.

**Comment by ggolcher** [comment]
M provides a counter-perspective:

"Gotta say, I think the lack of a like button and the requirement you must add text when you repost may be good things — that way the only way to respond to a post is to add a thought of your own, which encourages communication instead of virality."

**Reply by ramirolz** [reply]
In my opinion, giving feedback on someone else's posts is a must if we aim to create a valid tool of communication. The problem, in my opinion, is that the "like" button's feedback is too vague to be truly useful. Why did you like a particular post? Was it entertaining, funny, interesting, educational? For a feedback system to be efficient, we need more information and more precise. Closer to a tag system. May be more appealing if you can give medals or other sort of prizes with specific information. Like a medal for an educational post and another for a funny one. That way you could enable a search or a page layout in a particular community that would show you the posts with more "educational" points and in another community just get funny posts on your feed because that's what you are really interested in.
